src/main/java/pl/edu/pw/ee/catering/model/cateringcompany/service/impl/CateringCompanyImpl.java

```diff
@@ -31,4 +34,9 @@ public void editMeal(MealDetails mealDetails) {
     public void deleteMeal(Long id) {
         meal.deleteMeal(id);
     }
+
+    @Override
+    public OrderList showHistoricalOrderList(Long id) {
+        return new OrderList(new ArrayList<>()); // TODO: implement
```

Comment:
Tutaj TODOs znaleziony, to trzeba czeknąć czy czegoś nie brakuje.
Jeśli nie to chyba do wyrzucenia komentarz.

Reply:
Jak robiłem stronę frontendu to chciałem wtedy oznaczyć, że ten kod zostanie wywalony przy robieniu backendu w przyszłości przez osobę za to odpowiedzialną, tak to faktycznie komentarz w kodzie do wyrzucenia